Assets/MXR.SDK/Plugins/Android/AdminAppMessengerManager.java

```diff
@@ -115,6 +115,10 @@ private boolean registerAsClient() {
         return sendMessage(AdminAppMessageTypes.REGISTER_CLIENT);
     }
 
+    public boolean isAdminAppInstalled() {
```

Comment:
Should this live here? We aren't really sending a message here, are we?

Reply:
Ah yes, moved it to NativeUtils.java